CITATION.cff

```diff
@@ -31,10 +31,11 @@ keywords:
   - deep-learning
   - pytorch
   - image-generation
+  - hacktoberfest
   - diffusion
   - text2image
   - image2image
   - score-based-generative-modeling
   - stable-diffusion
+  - stable-diffusion-diffusers
 license: Apache-2.0
-version: 0.12.1
```

Comment:
Let's keep the version (maybe it can be updated)

Reply:
Hmm I guess it's not really a big deal if we remove the version then, since it hasn't been updated in a while. WDYT @sayakpaul?

Reply:
@sayakpaul, WDYT?

Reply:
Feel free to just leave it as is and we can update in the future if we need to :)

Reply:
Ok, reversed.